app/models/waste_carriers_engine/order_item_log.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
```

Comment:
This should probably be in the engine

Reply:
Agreed - I will remove it from back-office and use the one in the engine.